specification/resources/resource-manager/readme.python.md

```diff
@@ -25,7 +25,6 @@ batch:
   - tag: package-locks-2016-09
   - tag: package-locks-2015-01
   - tag: package-managedapplications-2018-06
-  - tag: package-managedapplications-2017-09
```

Comment:
why this tag is  removed?

Reply:
because managedapplications was intentionally generated as single-api, and the second tag was put by mistake here by service team.
basically, in current configuration **package-managedapplications-2017-09** will overwrite **package-managedapplications-2018-06** meaning that old package was released instead of new one.
so after discussing with @lmazuel I decided to leave it as single api and fix the problem introduced by the service team.